### PR TITLE
Update Implementation Report

### PR DESCRIPTION
**Pull request by mmccool**
- update template to correct policies, dates, etc
- number of small changes to text to align with what we actually did
- update testing contribution acknowledgements

### CODE REVIEW

File: testing/inputs/templates/report.html [hunk 1]
```diff
@@ -349,23 +319,29 @@ <h4 id="h_NotesOnTesting"><a id="NotesOnTesting" name="NotesOnTesting">5.2 Notes
...
 
 <h2 id="h_systems"><a id="test-systems" name="test-systems">6. Systems</a></h2>
 <p>
   This section contains descriptions of each of the implementations
-  of the WoT Thing Description specification
+  of the WoT Thing Description 1.1 specification
   from all organizations that submitted implementer reports.
   Each implementation represents a working system
-  that either exposes or consumes at least one WoT Thing Description.
+  that either provides a WoT Thing Description, consumes WoT Thing Descriptions,
```
Comment (mmccool) [inline testing/inputs/templates/report.html:343]:
```suggestion
  that either exposes (provides) a WoT Thing Description, consumes WoT Thing Descriptions,
```

**Review by mmccool — COMMENTED** [review]
some changes

File: testing/inputs/templates/report.html [hunk 2]
```diff
@@ -130,7 +130,7 @@ <h1 id="h_ir">
...
+  <p class="copyright"><a href="http://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © 2023 <a href="http://www.w3.org/"><acronym title="World Wide Web Consortium">W3C</acronym></a><sup>®</sup> (<a href="http://www.csail.mit.edu/"><acronym title="Massachusetts Institute of Technology">MIT</acronym></a>, <a href="http://www.ercim.org/"><acronym title="European Research Consortium for Informatics and Mathematics">ERCIM</acronym></a>, <a href="http://www.keio.ac.jp/">Keio</a>), All Rights Reserved. W3C <a href="http://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="http://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a href="http://www.w3.org/Consortium/Legal/copyright-documents">document use</a> rules apply.</p>
```
[the remark this answers is not in the document]
Comment (mmccool) [reply testing/inputs/templates/report.html:133]: will fix in upcoming commit

File: testing/inputs/templates/report.html [hunk 3]
```diff
@@ -130,7 +130,18 @@ <h1 id="h_ir">
     Fady Salama (Siemens / Technical University of Munich)
   </p>
 
-  <p class="copyright"><a href="http://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © 2022 <a href="http://www.w3.org/"><acronym title="World Wide Web Consortium">W3C</acronym></a><sup>®</sup> (<a href="http://www.csail.mit.edu/"><acronym title="Massachusetts Institute of Technology">MIT</acronym></a>, <a href="http://www.ercim.org/"><acronym title="European Research Consortium for Informatics and Mathematics">ERCIM</acronym></a>, <a href="http://www.keio.ac.jp/">Keio</a>), All Rights Reserved. W3C <a href="http://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="http://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a href="http://www.w3.org/Consortium/Legal/copyright-documents">document use</a> rules apply.</p>
+  <p class="copyright"> <!-- where is this class defined? -->
```
Comment (mmccool) [inline testing/inputs/templates/report.html:133]:
```suggestion
  <p class="copyright">
```

File: testing/report11.html [hunk 4]
```diff
@@ -128,7 +128,18 @@ <h1 id="h_ir">
     Fady Salama (Siemens / Technical University of Munich)
   </p>
 
-  <p class="copyright"><a href="http://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © 2022 <a href="http://www.w3.org/"><acronym title="World Wide Web Consortium">W3C</acronym></a><sup>®</sup> (<a href="http://www.csail.mit.edu/"><acronym title="Massachusetts Institute of Technology">MIT</acronym></a>, <a href="http://www.ercim.org/"><acronym title="European Research Consortium for Informatics and Mathematics">ERCIM</acronym></a>, <a href="http://www.keio.ac.jp/">Keio</a>), All Rights Reserved. W3C <a href="http://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="http://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a href="http://www.w3.org/Consortium/Legal/copyright-documents">document use</a> rules apply.</p>
+  <p class="copyright"> <!-- where is this class defined? -->
```
Comment (mmccool) [inline testing/report11.html:131]:
```suggestion
  <p class="copyright">
```

File: testing/report11.html [hunk 5]
```diff
@@ -7732,21 +7729,41 @@ <h1 id="h_appendix"><a id="appendix" name="appendix">Appendix</a></h1>
 		
 <h2 id="h_ack"><a id="ack" name="ack">Acknowledgements</a></h2>
 <p>
-  The Web of Things Working Group would like to acknowledge the contributions to the making of this document from the following individuals in various capacities.</p>
-<p>
-</p><ul>
+The Web of Things Working Group would like to acknowledge the contributions to 
+the making of this document from the following individuals in various capacities,
+including contributions to past testing inputs which were used for compatibility testing:</p>
+<ul>
```
Comment (egekorkan) [inline testing/report11.html:7735]: `Cristiano Aguzzi (Invited Expert)` is missing

**Review by egekorkan — DISMISSED** [review]

File: testing/report11.html (hunk 5 above)
Comment (egekorkan) [reply testing/report11.html:7735]: @chachamimm let us know if you have done contributions to the implementation report.

File: testing/report11.html [hunk 6]
```diff
@@ -7732,21 +7729,41 @@ <h1 id="h_appendix"><a id="appendix" name="appendix">Appendix</a></h1>
...
+<li>Luca Barbato (Luminem)</li>
```
Comment (mmccool) [inline testing/report11.html:7736]:
```suggestion
<li>Cristiano Aguzzi (Invited Expert)</li>
<li>Luca Barbato (Luminem)</li>
```

File: testing/inputs/templates/report.html [hunk 7]
```diff
@@ -590,21 +587,41 @@ <h1 id="h_appendix"><a id="appendix" name="appendix">Appendix</a></h1>
 		
 <h2 id="h_ack"><a id="ack" name="ack">Acknowledgements</a></h2>
 <p>
-  The Web of Things Working Group would like to acknowledge the contributions to the making of this document from the following individuals in various capacities.</p>
-<p>
+The Web of Things Working Group would like to acknowledge the contributions to 
+the making of this document from the following individuals in various capacities,
+including contributions to past testing inputs which were used for compatibility testing:</p>
 <ul>
+<li>Luca Barbato (Luminem)</li>
```
Comment (mmccool) [inline testing/inputs/templates/report.html:594]:
```suggestion
<li>Cristiano Aguzzi (Invited Expert)</li>
<li>Luca Barbato (Luminem)</li>
```

**Pull request merged**